test/cypress/integration/reusable/open-sdk-locales-data-test.js

```diff
@@ -0,0 +1,13 @@
+const localesData = require('../fixtures/locales-data.json')
+
+export function verifySdkLocalesData(iframeSelector) {
+  cy.getSdk(iframeSelector).then(sdk => {
+    expect(JSON.stringify(sdk.locales)).to.equal(JSON.stringify(localesData))
```

Comment:
`JSON.stringify` makes it hard to understand the mismatch between the expected and actual if there's an error.

Have you tried using `deep.equal`?

Reply:
Thank you Alex, fixed the tests and they work.